vars/buildImage.groovy

```diff
@@ -63,6 +63,17 @@ def call(params = [:]) {
                 // Showing logs in Jenkins is also a way
                 // to wait for the build to finsih
                 bc.logs('-f')
```

Comment:
Ahh OK, one thing that tripped me up here is that `startBuild` returns a "build selector" which selects the build which was started. The `bc` makes it seem like it's the `buildConfig` object itself.

Maybe let's rename that var to `build`?

Reply:
Thanks for the review @jlebon. Yes, I was able to test it forcing a synthesis error in cosa.
Done, changed it to build =)